tests/cases/tm2tm/scripts/test-service

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+: <<'END_COMMENT'
+* relay-interval = 20s
+* src-relay-optimize-interval = 25s
+* src-relay-optimize-count = 3
+* dst-relay-optimize-interval = 25s
+* dst-relay-optimize-count = 3
+
+- relay service [packets = 0, time = 0] -> skip
+    - sleep 4
+    - 
+    - transfer x 1
+    - packets = 1, ack = 0
+    - sleep 16
+    - 
+    - 
+    - 
+    - 
+- relay service [packets = 1, time = 0] -> skip
+    - sleep 4
+    - 
+    - transfer x 1
+    - packets = 2, ack = 0
+    - sleep 16
+    - 
+    - 
+    - 
+    - 
+- relay service [packets = 2, time = 20] -> skip
+    - sleep 20
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+- relay service [packets = 2, time = 20] -> exec (time)
+    - sleep 4
+    - 
+    - packets = 0, ack = 2
+    - transfer x 3
+    - packets = 3, ack = 2
+    - sleep 16
+    - 
+    - 
+    - 
+- relay service [packets = 3, time = 20] -> exec (count)
+    - sleep 4
+    - 
+    - packets = 0, ack = 5
+    - sleep 16
+    - 
+    - 
+    - 
+    - 
+    - 
+- relay service [packets = 0, time = 40] -> exec (ack) (time + count)
+    - sleep 4
+    - 
+    - packets = 0, ack = 0
+    - Finished
+END_COMMENT
+
+set -eux
+
+SCRIPT_DIR=$(cd $(dirname $0); pwd)
+RLY_BINARY=${SCRIPT_DIR}/../../../../build/yrly
+RLY="${RLY_BINARY} --debug"
+
+source ${SCRIPT_DIR}/utils
+
+TM_ADDRESS0=$(${RLY} tendermint keys show ibc0 testkey)
+TM_ADDRESS1=$(${RLY} tendermint keys show ibc1 testkey)
+
+${RLY} service start ibc01 --relay-interval 20s --src-relay-optimize-interval 25s --src-relay-optimize-count 3 --dst-relay-optimize-interval 25s --dst-relay-optimize-count 3 &
+RLY_PID=$!
+echo "xxxxxxx relay service [packets = 0, time = 0] -> skip xxxxxx"
+sleep 4
+
+# transfer a token
+${RLY} tx transfer ibc01 ibc0 ibc1 100samoleans ${TM_ADDRESS1}
+
+expectUnrelayedCount "unrelayed-packets" "src" 1
+expectUnrelayedCount "unrelayed-acknowledgements" "dst" 0
+sleep 16
+
+echo "xxxxxxx relay service [packets = 1, time = 0] -> skip xxxxxx"
+sleep 4
+
+${RLY} tx transfer ibc01 ibc0 ibc1 100samoleans ${TM_ADDRESS1}
+
+expectUnrelayedCount "unrelayed-packets" "src" 2
+expectUnrelayedCount "unrelayed-acknowledgements" "dst" 0
+sleep 16
+
+echo "xxxxxxx relay service [packets = 2, time = 10] -> skip xxxxxx"
```

Comment:
You changed the `relay-interval` parameter to 20, so `time = 20` is correct here?